src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -81,10 +84,17 @@
 TASK_END_TIME_KEY = 'task_end_time'
 
 POSTPROCESS_QUEUE = 'postprocess'
+UTASK_MAINS_QUEUE = 'utask_main'
 
 # See https://github.com/google/clusterfuzz/issues/3347 for usage
 SUBQUEUE_IDENTIFIER = ':'
 
+UTASK_TIME_LIMIT = 15
```

Comment:
Add a comment here describing what this does (and also add `_SECONDS`) to the constant name to make the unit clear.

Reply:
Made the  name more descriptive so as to not need a comment.